### PR TITLE
Tweaks: Fix "your tags" dropdown visual issue of Unsticky Tab Bar

### PR DESCRIPTION
**Pull request by AlecBlance**
### Description

Goal is to provide a fix in #968 by changing `position: static !important;` to `position: relative !important; top: 0 !important;`. 

### Testing steps

- Change `position: static !important;` to `position: relative !important; top: 0 !important;`.  https://github.com/AprilSylph/XKit-Rewritten/blob/c4a0a29d3aad83ab4dd49279fd0a9672f12907c2/src/scripts/tweaks/unsticky_tab_bar.js#L5
- Enable Unsticky Tab Bar tweak
- Go to your dashboard > your tags
- Click "filter by tags"

Confirm the fix works if it shows:

image

### CODE REVIEW

**Comment by AprilSylph** [comment]
(You don't need to list the changes in the PR in the testing steps, you can assume your reviewer will be running the branch you're pull-requesting)

**Comment by AprilSylph** [comment]
(also @marcustyphoon you know you're allowed to review PRs too, right? :D)

**Comment by AlecBlance** [comment]
> You don't need to list the changes in the PR in the testing steps, you can assume your reviewer will be running the branch you're pull-requesting

Thanks! I'm still not that familiar with github. I'll take note of this.

**Review by AprilSylph — CHANGES_REQUESTED** [review]
This does indeed fixed the referenced issue, but also causes a visual oddity where the tab bar will appear to jump down, and then slide back up when the tweak is first activated (or the page first loads). Overriding the `transition` property as well should remove this effect.

File: src/scripts/tweaks/unsticky_tab_bar.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { keyToCss } from '../../util/css_map.js';
 import { buildStyle } from '../../util/interface.js';
 
 const styleElement = buildStyle(`
-  ${keyToCss('tabsHeader')} { position: static !important; }
+  ${keyToCss('tabsHeader')} { position: relative !important; top: 0 !important;}
```
Comment (AprilSylph) [inline src/scripts/tweaks/unsticky_tab_bar.js:5]:
```suggestion
  ${keyToCss('tabsHeader')} {
    position: relative !important;
    top: 0 !important;
    transition: none !important;
  }
```

**Review by AprilSylph — COMMENTED** [review]
(Forgot to submit my suggestion before submitting my review...)

File: src/scripts/tweaks/unsticky_tab_bar.js (hunk 1 above)
Comment (marcustyphoon) [reply src/scripts/tweaks/unsticky_tab_bar.js:5]: See, this is why I didn't review this PR myself! :D

Comment (AlecBlance) [reply src/scripts/tweaks/unsticky_tab_bar.js]: I didn't notice this behavior. Thanks!

**Review by AprilSylph — APPROVED** [review]

**Pull request merged**